packages/terra-application-header-layout/src/terra-dev-site/doc/application-header-layout/ApplicationHeaderLayout.1.doc.mdx

```diff
@@ -0,0 +1,55 @@
+import { Badge } from '../../../../package.json?dev-site-package';
+
+import HeaderWireframe from '../example/HeaderWireframe?dev-site-example';
+
+import HeaderLayoutSrc from '../../../../src/ApplicationHeaderLayout.jsx?dev-site-props-table?dev-site-props-table';
+
+<Badge />
+
+# Terra Application Header Layout
```

Comment:
Do we need to delete the `docs/README.md`? It looks like this content has been copied over

Reply:
Deleted afd3e5e